src/components/Callout.tsx

```diff
@@ -66,21 +26,45 @@ const Callout: React.FC<IProps> = ({
   children,
   className,
 }) => (
-  <StyledCard className={className}>
-    {image && <Image image={image} alt={alt || ""} />}
-    <Content>
+  <Flex
+    direction="column"
+    bgGradient="linear-gradient(
+    49.21deg,
+    rgba(127, 127, 213, 0.2) 19.87%,
+    rgba(134, 168, 231, 0.2) 58.46%,
+    rgba(145, 234, 228, 0.2) 97.05%
+  )"
+    p={6}
+    m={4}
+    mt={32}
+    mb={{ base: 16, lg: 4 }}
+    borderRadius="base"
+    className={className}
+  >
+    {image && (
+      <Image
+        as={GatsbyImage}
+        image={image}
+        alt={alt || ""}
+        mt={-40}
+        alignSelf="center"
+        maxW="263px"
+        minH="200px"
+      />
+    )}
+    <Flex direction="column" justify="space-between" mt={10} h="full">
       <div>
-        {emoji && <Emoji text={emoji} size={3} />}
-        <h3>
+        {emoji && <Emoji text={emoji} fontSize="5xl" />}
+        <Heading as="h3" fontSize="2xl" lineHeight={1.4}>
```

Comment:
Nice. Soon I'll create the heading styles on Chakra theme so that we don't need to do this anymore. Just `<Heading as="h3">`